src/portable/ReactUi/react/src/components/TaskDetails.tsx

```diff
@@ -100,9 +103,9 @@ class TaskDetails extends React.Component<IProps, typeof initialState> {
                             <div className="taskItemContent">
                                 <div className="firstLine">
                                     <span className="displayReference">{reference}</span>
-                                    <span className={"totalTime" + (fileName !== ""? "": " hide")}>
-                                        <Duration seconds={this.duration()} direction={direction} />
-                                    </span>
+                                    <span className={"totalTime" + (fileName !== "" ? "" : " hide")}>
+                                        {durationWraper}
```

Comment:
This change seems to be included in the wrong commit. I think this change is related to TT-160